src/app/shared/model/feeder-episode.model.ts

```diff
@@ -52,6 +53,7 @@ export class FeederEpisodeModel extends BaseModel {
     this.authorName = author['name'] || '';
     this.authorEmail = author['email'] || '';
     this.summary = this.doc['summary'] || '';
+    this.enclosureUrl = this.doc.expand('enclosure') || '';
```

Comment:
👍 at least I feel better

Reply:
it was a good note - definitely that code was a cry for help.